### PR TITLE
[EM-18] create account screen

### PR DESCRIPTION
**Pull request by sukhi0077**

### CODE REVIEW

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+export const AggrementText = styled(Text)`
+  font-size: 14px;
+  margin-top: 30px;
+`;
+export const SignInLink= styled(Text)`
+  text-align: center;
+  font-size: 14px;
+  font-weight: bold;
+  color: #0717f7;
+`;
```
Comment (manpreet-tivix) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts:34]:
```suggestion
`;

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+export const SignInText = styled(Text)`
+  text-align: center;
+  font-size: 14px;
+  margin-top: 100px;
+`;
+
+export const AggrementText = styled(Text)`
+  font-size: 14px;
+  margin-top: 30px;
+`;
```
Comment (manpreet-tivix) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts:28]:
```suggestion
`;

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  border-radius: 10px;
+  background-color: #1b8ffa;
+  margin-top: 100px;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const SaveText = styled(Text)`
+  color: #FFFFFF;
+`;
```
Comment (manpreet-tivix) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts:54]:
```suggestion
`;

```

File: elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx [hunk 4]
```diff
@@ -7,23 +7,17 @@ import { PersonelInfoScreenProps } from './PersonelInfoScreen.interface';
 export const PersonelInfoScreen: FC<PersonelInfoScreenProps> = () => {
```
Comment (manpreet-tivix) [inline elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx:7]: personel => personal

File: elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx [hunk 5]
```diff
@@ -7,23 +7,17 @@ import { PersonelInfoScreenProps } from './PersonelInfoScreen.interface';
...
-      placeholder="Language" 
-    />
-      <SaveButton onPress={() => Alert.alert('Simple Button pressed')}>
-        <SaveText>SAVE</SaveText> 
-      </SaveButton> 
-    </View>
+      <View>
+        <TitleText>Personel info</TitleText>
+        <Input placeholder="Full Name"/>
+        <Input placeholder="Occocupation"/>
```
Comment (manpreet-tivix) [inline elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx:15]: Occocupation => Occupation

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.interface.ts [hunk 6]
```diff
@@ -0,0 +1,4 @@
+export interface CreateAccountScreenProps {
+  // TODO: add required props below
+  myProp?: boolean;
+}
```
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.interface.ts:4]:
```suggestion
}

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts (hunk 2 above)
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts:28]:
```suggestion
`;

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts (hunk 1 above)
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts:34]:
```suggestion
`;

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts (hunk 3 above)
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.styles.ts:54]:
```suggestion
`;

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.test.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+import { CreateAccountScreen } from './CreateAccountScreen';
+
+describe('containers/CreateAccountScreen', () => {
+  it('should render', () => {
+    const component = render(<CreateAccountScreen />);
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+});
```
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.test.tsx:10]:
```suggestion
});

```

File: elearning/src/containers/CreateAccountScreen/CreateAccountScreen.tsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        <Input placeholder="password" />
+        <AggrementText>I agree to terms and conditions</AggrementText>
+        <SaveButton onPress={() => Alert.alert('Simple Button pressed')}>
+          <SaveText>Create Account</SaveText>
+        </SaveButton>
+        <SignInText>Already have account? <SignInLink>Sign In</SignInLink></SignInText>
+      </View>
+    </Container>
+  );
+};
```
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/CreateAccountScreen.tsx:24]:
```suggestion
};

```

File: elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx [hunk 9]
```diff
@@ -7,23 +7,17 @@ import { PersonelInfoScreenProps } from './PersonelInfoScreen.interface';
...
+        <Input placeholder="Language"/>
+        <SaveButton onPress={() => Alert.alert('Simple Button pressed')}>
+          <SaveText>SAVE</SaveText>
+        </SaveButton>
+      </View>
     </Container>
   );
 };
```
Comment (mann579) [inline elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx:23]:
```suggestion
};

```

File: elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx (hunk 5 above)
Comment (mann579) [inline elearning/src/containers/PersonelInfoScreen/PersonelInfoScreen.tsx:15]:
```suggestion
        <Input placeholder="Occupation"/>
```

File: elearning/src/containers/CreateAccountScreen/index.ts [hunk 10]
```diff
@@ -0,0 +1 @@
+export { CreateAccountScreen } from './CreateAccountScreen';
```
Comment (mann579) [inline elearning/src/containers/CreateAccountScreen/index.ts:1]:
```suggestion
export { CreateAccountScreen } from './CreateAccountScreen';

```

**Pull request merged**